src/features/vars-schema/env-vars.ts

```diff
@@ -141,6 +141,34 @@ export const DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES: Array<{
 				description: 'Define device GPU memory in megabytes.',
 				default: 16,
 			},
+			BALENA_HOST_CONFIG_hdmi_cvt: {
+				type: 'string',
+				description: 'Define a custom CVT mode for the HDMI',
+				examples: ['480 360 60 1 0 0 0'],
+			},
+			BALENA_HOST_CONFIG_hdmi_force_hotplug: {
+				type: 'integer',
+				enum: [0, 1],
+				description: 'Force the HDMI hotplug signal',
+				default: 0,
+			},
+			BALENA_HOST_CONFIG_hdmi_group: {
+				type: 'integer',
+				description: 'Define the HDMI output group',
+				examples: [2],
+				default: 0,
+			},
+			BALENA_HOST_CONFIG_hdmi_mode: {
+				type: 'string',
```

Comment:
This could probably be `integer` unless you saw a possible value that I missed?

Reply:
Ooops you are right :sweat: thanks for spotting...
Is there a default value for this @klutchell ?

Reply:
`hdmi_mode=1` seems to be recommended to start and verify the display works, then increase it from there.

This field is ignored if `hdmi_group=0` so it shouldn't matter much what the default is here.